runner/runner.go

```diff
@@ -89,7 +89,6 @@ func (r *Runner) Run(ctx context.Context) error {
 			PlacementConstraints:    describeTaskDefinitionOutput.TaskDefinition.PlacementConstraints,
 			ProxyConfiguration:      describeTaskDefinitionOutput.TaskDefinition.ProxyConfiguration,
 			RequiresCompatibilities: describeTaskDefinitionOutput.TaskDefinition.RequiresCompatibilities,
-			Tags:                    describeTaskDefinitionOutput.Tags,
```

Comment:
Should we perhaps only add `Tags` if the output from the task definition isn't empty? eg empty array or nill?